pkgs/games/anki/addons/puppy-reinforcement/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  anki-utils,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+anki-utils.buildAnkiAddon (finalAttrs: {
+  pname = "puppy-reinforcement";
+  version = "1.1.1";
+  src = fetchFromGitHub {
+    owner = "glutanimate";
+    repo = "puppy-reinforcement";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-y52AjmYrFTcTwd4QAcJzK5R9wwxUSlvnN3C2O/r5cHk=";
+  };
+  sourceRoot = "${finalAttrs.src.name}/src/puppy_reinforcement";
+  passthru.updateScript = nix-update-script { };
+  meta = {
+    description = "Anki add-on that encourages learners with pictures of cute puppies";
```

```suggestion
    description = "Encourage learners with pictures of cute puppies";
```

It's under `ankiAddons`, so we already know that it's an Anki add-on.